tests/test_C0_data_processing.py

```diff
@@ -1080,7 +1085,37 @@ def test_change_sign_of_feedin_tariff_zero(caplog):
     ), f"A msg is logged although the feed-in tariff is not changed."
 
 
+def test_compute_timeseries_properties_TIMESERIES_in_dict_asset():
+    str = "str"
+    dict_asset = {
+        TIMESERIES: pd.Series([1, 2, 3]),
+        UNIT: "str",
+        LABEL: "str"
```

Comment:
Here you correctly defined `LABEL` directly for `dict_asset`.

Reply:
It worked! Thank you @smartie2076